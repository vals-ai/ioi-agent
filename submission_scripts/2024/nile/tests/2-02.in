version https://git-lfs.github.com/spec/v1
oid sha256:0b32e28aeda757d2415810c2e2efa92ce72060d728012925362173c8470d5080
size 2548931
