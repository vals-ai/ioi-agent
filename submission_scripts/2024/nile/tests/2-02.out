version https://git-lfs.github.com/spec/v1
oid sha256:867cb7f6bfee8b6cf95f23aebb16a035efdb446d6c85a68e664dcfcaaaad39b1
size 111
